config/options.yml

```diff
@@ -0,0 +1,13 @@
+shared:
+  features:
+    voice: <%= ENV.fetch("VOICE_FEATURE", false) %>
+    registration: <%= ENV.fetch("REGISTRATION_FEATURE", true) %>
+    password_authentication: <%= ENV.fetch("PASSWORD_AUTHENTICATION_FEATURE", true) %>
+    google_authentication: <%= ENV.fetch("GOOGLE_AUTHENTICATION_FEATURE", true) %>
+    http_header_authentication: <%= ENV.fetch("HTTP_HEADER_AUTHENTICATION_FEATURE", true) %>
+  settings:
+    google_auth_client_id: <%= ENV["GOOGLE_AUTH_CLIENT_ID"] || Rails.application.credentials[:google_client_id] %>
+    google_auth_client_secret: <%= ENV["GOOGLE_AUTH_CLIENT_SECRET"] || Rails.application.credentials[:google_client_secret] %>
+    http_header_auth_email: <%= ENV["HTTP_HEADER_AUTH_EMAIL"] || ENV["X-WEBAUTH-EMAIL"] %>
```

Comment:
I am ok with `HTTP_HEADER_AUTH_EMAIL `.

However, `ENV["X-WEBAUTH-EMAIL"]` would not fit. `X-WEBAUTH-EMAIL` should be the default value for the variable `http_header_auth_email`.

Therefore: `<%= ENV.fetch("HTTP_HEADER_AUTH_EMAIL", 'X-WEBAUTH-EMAIL') %>`